ui/src/app/base/components/FormikFormContent/FormikFormContent.test.tsx

```diff
@@ -268,4 +269,56 @@ describe("FormikFormContent", () => {
     wrapper.update();
     expect(wrapper.find("input[name='val1']").props().value).toBe("initial");
   });
+
+  it("runs onSuccess function if successfully saved with no errors", async () => {
+    const onSuccess = jest.fn();
+    const store = mockStore(state);
+
+    const Proxy = ({ saved }: { saved: boolean }) => (
```

Comment:
This `Proxy` component is used as a workaround to set component props on components connected to the store, since you can only set props on the top-level component (which would otherwise be `Provider`).